salt/modules/reg.py

```diff
@@ -76,13 +118,18 @@ def broadcast_change():
     '''
     Refresh the windows environment.
 
-    Returns (bool): True if successful, otherwise False
+    .. note::
+        This will only effect new processes and windows. Services will not see
+        the change until the system restarts.
```

Comment:
until the service is restarted.

Reply:
@damon-atkins Any services running under the Local System account will not receive the new environment until you restart.
https://support.microsoft.com/en-us/help/821761/changes-that-you-make-to-environment-variables-do-not-affect-services

Reply:
un-expected, but makes senses,